designer/server/lib/persistence/previewPersistenceService.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import type { PersistenceService } from './persistenceService'
+const Wreck = require('@hapi/wreck')
+const config = require('../../../config')
+
+export class PreviewPersistenceService implements PersistenceService {
```

Comment:
I think this should be documented/mentioned this is intended to be used with the runner

Reply:
also possibly in the runner it should be mentioned that changing the endpoints for publish might affect the designer

Reply:
Ok